### ISSUE TITLE
Information about the language of the implementations would be helpful

### ISSUE THREAD

**Issue by Vampire** [fingolfin/ssh-comparison#77]
If I'm for example searching a Java implementation that supports a specific extension, it would be nice to be able to find out or even filter for the language of an implementation if it is a library.

**Comment by fingolfin** [comment]
Yeah, filtering would be nice, see also https://github.com/fingolfin/ssh-comparison/issues/6 (but these days I have little time to work on this hobby, so somebody else will have to contribute this, I am afraid).

And adding the implementation language where known is of course also fine.

**Comment by Vampire** [comment]
Ah, ok, then #6 for the filtering and here for adding language. :-)
